### PR TITLE
plugins/ocp: Add DSSD Power state feature(FID: C7h) 

### PR DESCRIPTION
**Pull request by Arunpandian15**
Added DSSD Power state feature(FID: C7h) in OCP plugin and related documentation, bash completion in respective directory.

### CODE REVIEW

File: plugins/ocp/ocp-nvme.c [hunk 1]
```diff
@@ -1863,6 +1863,103 @@ static int ocp_device_capabilities_log(int argc, char **argv, struct command *cm
...
+///////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////
+/// DSSD Power State (Feature Identifier C7h) Set Feature
+
+static int set_dssd_power_state(struct nvme_dev *dev, const __u32 nsid,
+								const __u8 fid, __u8 power_state, bool save,
+								bool uuid)
+{
```
Comment (igaw) [inline plugins/ocp/ocp-nvme.c:1875]: The indention looks a bit off.

**Comment by igaw** [comment]
Just rebased to fix CI builds.

**Comment by igaw** [comment]
@arthurshau could you review this change? Thanks!

**Comment by arthurshau** [comment]
Yeah, LGTM

File: plugins/ocp/ocp-nvme.c (hunk 1 above)
Comment (igaw) [reply plugins/ocp/ocp-nvme.c:1875]: Fix it myself.

**Pull request merged**